Src/Sarif.PatternMatcher/AnalyzeContext.cs

```diff
@@ -45,7 +45,7 @@ public AnalyzeContext()
 
         public string GlobalFileDenyRegex { get; set; }
 
-        public int MaxFileSizeInKilobytes { get; set; }
+        public int MaxFileSizeInKilobytes { get; set; } = 1024 * 10000;
```

Comment:
Isn't this math wrong?  ~10mb in kb should either be 1024 * 10, or just ~10,000.

Reply:
Yes.